index.js

```diff
@@ -72,7 +72,7 @@ module.exports = {
       minifyJS: this.app.options.minifyJS,
       fingerprint: this.app.options.fingerprint.enabled,
```

Comment:
I am not sure but I think this might throw an error if the fingerprint option is not defined in the application?

Reply:
It is defaulted in the `included` hook.